### PR TITLE
Use new version

### PR DESCRIPTION
**Pull request by wachsylon**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by wachsylon** [comment]
@conda-forge-admin, please rerender

**Comment by wachsylon** [comment]
Hi,
I tried an easy update but failed...
At first, the installation failed because there was no MacOS version 10 where the CI could start from. I thought that this was caused by a missing rerendering so I commented here as recommended. After 10 minutes without reactions by the conda-forge admin, I did it locally by myself. I pushed it and now I got lib conflicts which I do not understand, e.g.:
```
Package libuuid conflicts for:
libuuid=2.32.1
cmor -> libuuid[version='>=2.32.1,<3.0a0']
```
If a bigger equal sign is what I think that it is, then - where is the conflict? :)

What did I do wrong?

Another question: Do I need to increase the build number if the version of CDO remains 1.9.9 but the rc increases?

Best regards,
Fabi

**Comment by Try2Code** [comment]
hi @wachsylon !
I successfully build your fork, but I have to clue how to work with the versioning, sorry.

And I think, build numbers should always be increased if something will be re-rendered

**Comment by ocefpaf** [comment]
We need to change proj4 to proj and rerender. But I guess we may need new builds of https://github.com/conda-forge/json-c-feedstock/pull/14 and cmor.

**Comment by wachsylon** [comment]
> We need to change proj4 to proj and rerender
I think I did that now.

> But I guess we may need new builds of conda-forge/json-c-feedstock#14 and cmor.
So if I remove them from build it should work? I may try that just for testing, O.K.?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ requirements:
   host:
     - jasper
     - libnetcdf
-    - proj4
+    - proj
```
Comment (ocefpaf) [inline recipe/meta.yaml:25]: Must changed it below too.

**Comment by ocefpaf** [comment]
> So if I remove them from build it should work? I may try that just for testing, O.K.?

Worth testing. Let's see if that gets us further.

**Comment by wachsylon** [comment]
@ocefpaf You were right. Without `CMOR` and with `proj` it works.  What do you mean with

> new builds of conda-forge/json-c-feedstock#14 and cmor.

Do they just need to rerender? Can we create an issue?
I assume this takes a while. Can you accept the pull request for now and if cmor can be linked again I rechange it in a new pull request?

**Comment by ocefpaf** [comment]
> Do they just need to rerender? Can we create an issue?

I there is a PR with that already and it is failing. We need someone who has time and some knowledge of `json-c` to fix it.

> I assume this takes a while. Can you accept the pull request for now and if cmor can be linked again I rechange it in a new pull request?

Sure. In fact that is more up to you. I'm not a user of `cdo` or `cmor` :smile: 

Merge away if you are ready with this PR!

**Comment by wachsylon** [comment]
> Merge away if you are ready with this PR!

@Try2Code Can you do that? I think you also need to do that review. It says it is pending

**Comment by ocefpaf** [comment]
> @Try2Code Can you do that? I think you also need to do that review. It says it is pending

That is boilerplate GH stuff. No need to worry about it :smile:

**Comment by Try2Code** [comment]
here you go ;-)

**Comment by djsutherland** [comment]
FYI @wachsylon @conda-forge/cdo, new builds of `json-c` are about to be up (conda-forge/json-c-feedstock#15, which included conda-forge/json-c-feedstock#14).

**Pull request merged**